sources/Context.cpp

```diff
@@ -108,9 +132,237 @@ Context::process() {
   onNodeOnMainConnexComponent(algo::ControllerInterfaceDefinitionAlgorithm(hvdcLines_));
   walkNodesMain();
 
+  // Check all contingencies have their elements present in the network and make
+  // a new list of Contingencies with only those that are good
```

Comment:
comment: 'with only those that are acceptable'

Reply:
Ok